requirements.txt

```diff
@@ -6,4 +6,4 @@ requests
 typing-extensions
 # Newer version does not work with default OpenSSL version on jammy.
 cryptography <= 38.0.4
-pydantic>=1,<2
\ No newline at end of file
+pydantic >= 1.10.13,<2
```

Comment:
Sorry, I meant to the exact version, so that we know what's being pulled and let renovate handle the updates

Reply:
Yes, I assumed that, I added a comment above https://github.com/canonical/github-runner-operator/pull/127#discussion_r1339971396. As it would be the first package to be pinned in this charm, I asked @yhaliaw for input as it may be that the packages are not pinned for some reason/design choice.

Reply:
I'd say consistant in the pinning approach, but we should definitely have a global discussion within the team on pinning

Reply:
I have now pinned it.